src/i18n/locales/zh-CN

```diff
@@ -0,0 +1,375 @@
+  /* I have an issue with my v3.8.1. After clicking Connect to Pocket, I just got a blank page on my Mac
```

Comment:
Hmm, that's not good - what version of MacOS are you using?

Reply:
Ventura 13.5.2
Today, the main page worked. No clue what's the cause...

Reply:
Yeah that should be supported, weird - not sure what'd cause that, glad it's working now though